snooty.toml

```diff
@@ -16,16 +16,17 @@ toc_landing_pages = [
 sharedinclude_root = "https://raw.githubusercontent.com/10gen/docs-shared/main/"
 
 [constants]
-version = "4.7"
+version = "4.10"
 package-name-org = "mongodb-org"
 api = "https://mongodb.github.io/node-mongodb-native/{+version+}"
 mongosh = "``mongosh``"
 driver = "node"
 driver-long = "MongoDB Node.js driver"
 driver-short = "Node.js driver"
+node-legacy = "`MongoDB Node.js Driver with Optional Callback Support <https://www.npmjs.com/package/mongodb-legacy>`__"
```

Comment:
Our documentation seems inconsistent with regard to capitalizing "driver." Just checking that these three lines are the way you think they should be.